tests/data/jsons/file_types.json

```
@@ -6,14 +6,15 @@
         {
             "name": "FileTypeAssociation",
             "description": "Testing file type association",
-            "icon": null,
+	    "icon": null,
             "command": [
                 "{{ PYTHON }}",
                 "-c",
                 "import sys, pathlib as p; p.Path(r'__OUTPUT_FILE__').write_text(sys.argv[1])"
             ],
             "platforms": {
                 "win": {
+                    "icon": "icon.{{ ICON_EXT }}",
```

Where's this icon? 🤔 Should we add one?

---

Technically not. Whether the icon exists is never verified and it doesn't need to exist to successfully add the registry entry.

---

Let's name it something else so this is clear. Something like this:

```suggestion
                    "icon": "doesnotexistbutitsok.{{ ICON_EXT }}",
```